content/docs/06-integrations/00-spectro-k8s-dashboard.md

```diff
@@ -0,0 +1,144 @@
+
+# Versions Supported
+
+<Tabs>
+
+<Tabs.TabPane tab="2.6.x" key="2.6.x">
+
+## Prerequisites
+
+- Outbound internet connectivity for port 443 is allowed so that you and your applications can connect with the Spectro Cloud reverse proxy.
+
+## Parameters
+
+The Spectro Kubernetes Dashboard supports the following parameters. 
+<br />
+
+#### User Selections
+
+These settings are also included in the manifest as `access` and `identityProvider` parameters.
+
+| Parameter | Description | Default |
+|-----------|-------------|---------|
+| Access | Controls cluster access. Private access means that nodes and pods are isolated from the internet. | Private |
+| Identity Provider | You can use Palette as the IDP or a third-party application as the IDP. | Palette |
+
+#### Internal Manifest Parameters
+
+| Parameter | Description | Default |
+|-----------|-------------|---------|
+| namespace | The Kubernetes namespace to install the dashboard. | `kubernetes-dashboard` |
+| ClusterRole | The ClusterRole to assign to the Kubernetes Dashboard. | `read-only` |
+| certDuration | Self-signed certificate duration in hours. | 8760h (365d) |
+| certRenewal | Self-signed certificate renewal in hours | 720h (30d) |
+| enableInsecureLogin | RBAC ensures secure login. | `false` |
+| serviceType | The service type for the dashboard. Supported values are ClusterIP, LoadBalancer, and NodePort. | `ClusterIP` |
+| skipLogin | Enables or disables the skip login option on the dashboard. | `false` |
+| enableInsecureLogin | Enables non-Secure Sockets Layer (SSL) login. Dashboard login is always restricted to HTTP(S) + localhost or HTTPS and external domain. | `false` |
+| ingress:enabled | Ingress configuration to access the ClusterIP, loadBalancer, or nodePort. | `false` |
+
+## Usage
+
+To use the Spectro Kubernetes Dashboard pack, you have to add it to your cluster profile. The Spectro Kubernetes Dashboard pack has default settings for **Access** and **Identity Provider**. These settings automatically install the latest version of the Spectro Proxy pack when the cluster is created.
+
+Changing these default settings may require some additional configuration.
+<br /> 
+
+### Access
+
+The default setting is **Private**. If you change the setting to **Public** and your cluster is in a public cloud, there is no additional configuration.  
+
+However, if you change **Access** to **Public** and your cluster is in a private cloud, you have to manually add and configure the Spectro Proxy pack to your cluster profile. For more information, refer to the [Spectro Proxy](https://docs.spectrocloud.com/integrations/frp) guide.
+<br /> 
+
+### Identity Provider 
+
+The default setting is **Palette**. If you change this setting to **Inherit from Tenant**, you must configure OpenID Connect (OIDC) authentication. You can configure OIDC two ways:
+
+- In Tenant Admin scope, navigate to **Tenant Settings > SSO**, choose **OIDC**, and provide your third-party Identity Provider details.  
```

Comment:
Let's link to the Palette SSO page. https://docs.spectrocloud.com/user-management/saml-sso

Reply:
Added link.